src/coreclr/vm/zapsig.cpp

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+ // Licensed to the .NET Foundation under one or more agreements.
```

Comment:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
```

Reply:
Yeah, that change was only in there to make sure the build would trigger at least the coreclr stuff to run. As it appears it triggers everything anyways, I've reverted it.